uqcsbot/scripts/wolfram.py

```diff
@@ -0,0 +1,192 @@
+from uqcsbot import bot, Command
+from typing import Iterable, Tuple
+import requests
+import json
+
+APP_ID = "UVKV2V-5XW2TETT69"
+
+# IDEA: Show the assumptions made?
+# TODO: Better naming of commands
+@bot.on_command("wolframfull")
+async def handle_wolframfull(command: Command):
```

Comment:
maybe instead of a new command, you can handle an optional variable? looks like it would remove some code duplication as well.

i.e. !wolfram something and then !wolfram something --full

if this is too annoying, consider it an optional improvement :^)

Reply:
lmk if you think they're sufficiently different responses that it warrants splitting

Reply:
I was on the fence about this too. I was concerned that the usual  "- - cmd" would interfere with math input but mainly that they do do pretty different things.